src/riscv/lib/src/jit/builder/sequence.rs

```diff
@@ -64,8 +68,14 @@ pub struct SequenceBuilder<'jit, MC: MemoryConfig> {
     /// Offset to the program counter for the next instruction
     program_counter_offset: i64,
 
+    /// Initial maximum number of steps that can be executed in the sequence
+    max_steps: Value<u64>,
```

Comment:
All the other parameter fields have the `_param` suffix. Not sure if the suffix is really needed but consistency would be valueable nevertheless, I think.